agent/http_register.go

```diff
@@ -94,6 +94,7 @@ func init() {
 	registerEndpoint("/v1/internal/ui/nodes", []string{"GET"}, (*HTTPServer).UINodes)
 	registerEndpoint("/v1/internal/ui/node/", []string{"GET"}, (*HTTPServer).UINodeInfo)
 	registerEndpoint("/v1/internal/ui/services", []string{"GET"}, (*HTTPServer).UIServices)
+	registerEndpoint("/v1/internal/ui/gateway-services-nodes", []string{"GET"}, (*HTTPServer).UIGatewayServicesNodes)
```

Comment:
I think you need to add a trailing `/` here, not exactly sure why but without it I could not get this endpoint to work

Reply:
fixed